### ISSUE TITLE
Netplay stuck on "Starting..."

### ISSUE THREAD

**Issue by radiohazard-dev** [mmatyas/supermariowar#199]
I've been trying to get `smw-server` to work for quite some time, and it works just fine for LAN play. However, this particular setup doesn't seem to work.

I have the server set up and exposed to the public (12521/UDP,) and I have two users: one on LAN (me) and one from outside my network. I'm able to create a room, and we're both able to join it, but pressing "Start" just hangs. Nothing changes on their end. I've confirmed that the server and the client have the same versions: the latest `continuous` commit.

The full console log is in [this Pastebin](https://pastebin.com/mJbWmUv9). 
The `serverlog.txt` is in [this one](https://pastebin.com/c3mzrNJ8).

(The other user may not be receiving the game start message due to the Windows Firewall, but they say they've turned it off.)

**Comment by mmatyas** [comment]
Could you try also forwarding the 1252**2** UDP port for _both_ players? This is where the game clients communicate with each other. (Yeah, this part is indeed not documented)

**Comment by 2Shad** [comment]
hold on, do the players need to be portforward 12522 themselves as well? as i have port 12521-12522 forwarded on server side, but same issue

**Comment by mmatyas** [comment]
Yes, the actual gameplay happens between the players at the moment, so in some cases you might need to port forward for the players too. Alternatively, if only the server player creates the rooms, that should also work.

**Comment by boostedgti916** [comment]
Can we have some more help on this? We opened ports on both sides but game is stuck on Starting....
